portal-api/api-tests/v1/users/login-with-sign-in-link.api-test.js

```diff
@@ -300,15 +300,15 @@ describe('POST /users/:userId/sign-in-link/:signInToken/login', () => {
             });
           });
 
-          it('returns a user blocked 403 error', async () => {
+          it('returns a user disabled 403 error', async () => {
```

Comment:
I updated the API to send the correct disabled or blocked error for the user, the frontend as per initial requirements still shows the blocked page for either a disabled or blocked user